libs/dh/metering-point/feature-child-overview/src/lib/dh-metering-point-child-overview.component.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ChangeDetectionStrategy, Component, NgModule } from '@angular/core';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'dh-metering-point-child-overview',
+  styles: [
+    `
+      :host {
+        display: block;
+      }
+    `,
+  ],
```

Comment:
question: do you want to use external file styles, inline styles, or it depends?

Reply:
I would suggest that we aim to keep it inline, unless a lot's of styles is necessary

Reply:
Can we normalize this across our components now while the number of components is low?